### ISSUE TITLE
Bfruit, Double Cross v.2.0, and Zelda Picross do not run.

### ISSUE THREAD

**Issue by AchillesPDX** [OnionUI/Ports-Collection#6]
Running the latest 4.2 Beta of OnionOS and extracted the Ports collection v1.0.6. All of the other ports included work correctly, but these 3 don't launch and instead crash back to the game list.

**Comment by TotallyNotSethP** [comment]
Could reproduce on my Miyoo Mini+

**Comment by RetroBreeze** [comment]
I was able to reproduce this on Miyoo Mini Plus as well.

**Comment by schmurtzm** [comment]
That's right!
It seems that it was ok on Onion 4.1 and not with Onion 4.2.

The libraries and the order in which they were loaded can have a big impact.

I'll check that tonight 😉

**Comment by TotallyNotSethP** [comment]
Thanks @schmurtzm :D

**Comment by schmurtzm** [comment]
OK, 
There ports are crashing : 
```
Fatal Python error: (pygame parachute) Segmentation Fault
Aborted (core dumped)
```

Now I know that it comes from the updated libraries and binaries from Miyoo.
If you replace  sdcard/miyoo folder by an old one it's working.

The issues seems to be related to sound output (and more precisely to audioserver) , strace result :  
`stat64("/dev/sound", 0xbe889168)        = -1 ENOENT (No such file or directory)`

For now I don't have a solution to solve that 😅
May be an ugly trick could be to run the old audioserver temporarily 😒
It will require more investigation ...

**Comment by schmurtzm** [comment]
This could be a good inspiration to load a previous version of audioserver (not totally sure that it is the solution but it deserve a try) : 
https://github.com/Rparadise-Team/Koriki/commit/2aae3f56c7973437939f1d71bfe83e0f70b0b486

**Comment by schmurtzm** [comment]
All are now fixed, it was a problem due to the sound management. The fixes will be published in the Onion ports v1.0.8

**Comment by schmurtzm** [comment]
Fixed in Onion ports v1.0.8 : 
https://github.com/OnionUI/Ports-Collection/commit/57eb5b0d4ce156730aec46abdf251c4a51d99f63
https://github.com/OnionUI/Ports-Collection/commit/9c756e27501f80382db7528b7f0101eba83326ce
